src/Toolbar/ToolbarGroup.js

```diff
@@ -124,8 +124,7 @@ class ToolbarGroup extends Component {
         case 'DropDownMenu' :
           return React.cloneElement(currentChild, {
             style: Object.assign({}, styles.dropDownMenu.root, currentChild.props.style),
-            styleControlBg: styles.dropDownMenu.controlBg,
-            styleUnderline: styles.dropDownMenu.underline,
+            underlineStyle: styles.dropDownMenu.underline,
```

Comment:
FYI, these were both previously not doing anything... and not sure what on eath `styleControlBg` was supposed to be.